azure-quantum/azure/quantum/qiskit/backends/qci.py

```diff
@@ -119,7 +118,6 @@ def __init__(self, name: str, provider: "AzureQuantumProvider", **kwargs):
                 "open_pulse": False,
                 "gates": [{"name": "TODO", "parameters": [], "qasm_def": "TODO"}],
                 "azure": self._azure_config(),
-                "is_default": True,
```

Comment:
I think this is not correct either. This should be the default backend for QCI since for this provider QIR is the only option. Changes to this file are not really necessary.